src/declarative-stack.ts

```diff
@@ -35,7 +34,7 @@ export class DeclarativeStack extends cdk.Stack {
 
     const schema = renderFullSchema(typeSystem);
 
-    const result = jsonschema.validate(template, schema);
+    const result = jsonschema.validate(template.template, schema);
```

Comment:
Is this still useful?

Reply:
@otaviomacedo I don't think so. Parsing and evaluating will also cover these cases.

Current tests pass without the validation. However we also don't have any tests for the schema rendering right now. I think addressing these issues, refactoring and adding tests in a separate PR would be the way to go here.